### ISSUE TITLE
[bug]  project init error

### ISSUE THREAD

**Issue by Hugood** [tauri-apps/tauri#8794]
### Describe the bug

When I init and run a new project use `pnpm create tauri-app`   and  `pnpm tauri dev` , a problem occurs like below, How can I solve?

### Full `tauri info` output

```text
- OS: Mac OS 14.2.1 X64
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.58.1 (db9d1b20b 2022-01-20)
    ✔ cargo: 1.58.0 (f01b232bc 2022-01-19)
    ✔ rustup: 1.24.3 (ce5817a94 2021-05-31)
    ✔ Rust toolchain: stable-x86_64-apple-darwin (default)
    - node: 18.19.0
    - pnpm: 7.33.6
    - yarn: 1.22.17
    - npm: 10.2.3
```

### Stack trace

```text
error: failed to select a version for the requirement `serde_with = "^3"`
candidate versions found which didn't match: 1.14.0, 1.13.0, 1.12.1, ...
location searched: crates.io index
required by package `tauri-utils v1.5.0`
    ... which satisfies dependency `tauri-utils = "^1.5.0"` of package `tauri-build v1.5.0`
    ... which satisfies dependency `tauri-build = "^1.5"` of package `tauri-app v0.0.0 (/Users/ashuai/work/tauriProject/tauri-app/src-tauri)`
 ELIFECYCLE  Command failed with exit code 101.
ashuai@localhost tauri-app % cargo tauri info
```

**Comment by FabianLars** [comment]
Can you try updating your rust/cargo version first? 1.58 is too old to build a tauri app and most of its dependencies anyways, but i hope it also causes this problem.

**Comment by Hugood** [comment]
This problem is solved

> Can you try updating your rust/cargo version first? 1.58 is too old to build a tauri app and most of its dependencies anyways, but i hope it also causes this problem.

Thank you，the  problem is solved，I update rust and cargo to 1.75.0
